rotkehlchen/chain/ethereum/transactions.py

```diff
@@ -148,6 +158,14 @@ def _get_transactions_for_range(
         dbethtx = DBEthTx(self.database)
         for query_start_ts, query_end_ts in ranges_to_query:
             log.debug(f'Querying Transactions -> {query_start_ts} - {query_end_ts}')
+            self.msg_aggregator.add_message(
```

Comment:
here and elsewhere remove the log.debug messages. The UI should be enough to check these from. And in tests or debugging even when there is no websockets/notifier it should be added to the legacy errors of the `msg_aggregator`